### PR TITLE
INC-1039: source Renovate config from shared HMPPS repo

### PR DESCRIPTION
**Pull request by neilmendum**
https://dsdmoj.atlassian.net/browse/INC-1039

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -1,38 +1,13 @@
...
-    ":preserveSemverRanges"
-  ],
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["github>ministryofjustice/hmpps-renovate-config:jvm"],
   "assigneesFromCodeOwners": true,
-  "timezone": "Europe/London",
-  "schedule": [
-    "after 9am every weekday",
-    "before 5pm every weekday"
-  ],
```
Comment (neilmendum) [inline renovate.json:12]: We now inherit this from https://github.com/ministryofjustice/hmpps-renovate-config/blob/4acc8951af908562c507a76b68ead835193c6a31/base.json#L8-L12

File: renovate.json [hunk 2]
```diff
@@ -1,38 +1,13 @@
...
-  "enabledManagers": ["gradle", "gradle-wrapper", "dockerfile", "docker-compose", "helmv3", "helm-values", "circleci"],
```
Comment (neilmendum) [inline renovate.json:13]: We now inherit this from https://github.com/ministryofjustice/hmpps-renovate-config/blob/4acc8951af908562c507a76b68ead835193c6a31/jvm.json#L4

File: renovate.json [hunk 3]
```diff
@@ -1,38 +1,13 @@
...
   "packageRules": [
     {
-      "matchDatasources": ["docker"],
-      "allowedVersions": "19-jre-jammy"
```
Comment (neilmendum) [inline renovate.json:17]: This is now pinned in https://github.com/ministryofjustice/hmpps-renovate-config/blob/4acc8951af908562c507a76b68ead835193c6a31/jvm.json#L7-L8

File: renovate.json [hunk 4]
```diff
@@ -1,38 +1,13 @@
...
-    },
-    {
-      "groupName": "kotlin plugins",
-      "matchPackagePrefixes": [
-        "org.jetbrains.kotlin.plugin"
-      ]
+      "matchManagers": ["gradle"],
+      "matchUpdateTypes": ["minor", "patch"],
+      "groupName": "all non major Gradle dependencies",
+      "groupSlug": "all-npm-minor-patch"
```
Comment (neilmendum) [inline renovate.json:10]: Group _all_ minor/patch gradle updates together in a rather than just a few packages

**Review by xoen — APPROVED** [review]

**Pull request merged**